sonar-slang-plugin/src/main/java/com/sonarsource/slang/kotlin/ChecksVisitor.java

```diff
@@ -94,6 +98,11 @@ public void reportIssue(Tree tree, String message, List<SecondaryLocation> secon
       reportIssue(tree.metaData().textRange(), message, secondaryLocations, gap);
     }
 
+    @Override
+    public String fileContent() {
+      return inputFileContextExtractor.content(currentCtx.inputFile);
```

Comment:
`InputFileContentExtractor` was useful when we wanted to support older versions of SQ (before SQ 6.2). I don't see why we need it now.